### PR TITLE
Fixed merge issues with climate demo

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Review by karolkolodziej — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

File: samples/dashboards/demo/climate/demo.html [hunk 1]
```diff
@@ -6,12 +6,12 @@
 <script src="https://code.highcharts.com/highcharts.js"></script>
 <script src="https://code.highcharts.com/highcharts-more.js"></script>
 <script src="https://code.highcharts.com/modules/accessibility.js"></script>
-<script src="https://code.highcharts.com/modules/data-tools.js"></script> <!-- @todo replace with math module -->
 <script src="https://code.highcharts.com/modules/map.js"></script>
```
Comment (goransle) [inline samples/dashboards/demo/climate/demo.html:9]: want to fix this also?

**Review by goransle — APPROVED** [review]

File: samples/dashboards/demo/climate/demo.html (hunk 1 above)
Comment (sebastianbochan) [reply samples/dashboards/demo/climate/demo.html:9]: Is it not included in the math-modifier.js ?

Comment (karolkolodziej) [reply samples/dashboards/demo/climate/demo.html:9]: I think @goransle meant `modules/map`, right?

Comment (sebastianbochan) [reply samples/dashboards/demo/climate/demo.html:9]: Mea culpa, you are right!

Comment (bre1470) [reply samples/dashboards/demo/climate/demo.html]: Thanks, fixed map & stock.

**Review by pawellysy — APPROVED** [review]

**Pull request merged**